tests/jballerina-unit-test/src/test/resources/test-src/object/object_initializer_negative.bal

```diff
@@ -92,3 +94,17 @@ type Person4 object {
         return e;
     }
 };
+
+type Too object {
+    public function __init() {
+        io:println("__init");
```

Comment:
We don't need this for the test right? We usually try to minimize imports in language unit tests. IMO, we can have an empty method here.

Reply:
Removed the print statement and import.